gubernator/view_pr_test.py

```diff
@@ -16,18 +16,48 @@
 
 import datetime
 
-from webapp2_extras import securecookie
-
+# TODO(fejta): use non-relative imports
+# https://google.github.io/styleguide/pyguide.html?showone=Packages#Packages
```

Comment:
"All new code should import each module by its full package name."

no need to change!